### PR TITLE
cleanup: remove fmt and errors new err

### PR DESCRIPTION
**Pull request by demoManito**
#### Description (what this PR does / why we need it):

static err should use `errors.New`, the performance will be higher than `fmt.Errorf`

reference: https://github.com/uber-go/guide/blob/master/style.md#errors

### CODE REVIEW

**Review by haiyux — APPROVED** [review]

File: contrib/registry/nacos/registry.go [hunk 1]
```diff
@@ -82,7 +83,7 @@ func New(cli naming_client.INamingClient, opts ...Option) (r *Registry) {
 // Register the registration.
 func (r *Registry) Register(_ context.Context, si *registry.ServiceInstance) error {
 	if si.Name == "" {
-		return fmt.Errorf("kratos/nacos: serviceInstance.name can not be empty")
+		return errors.New("kratos/nacos: serviceInstance.name can not be empty")
```
Comment (Casper-Mars) [inline contrib/registry/nacos/registry.go:86]: Would it be better to declare as a variable?

Comment (demoManito) [reply contrib/registry/nacos/registry.go:86]: What does it mean to 'declare as a variable'? means `fmt.error` better?

Comment (Casper-Mars) [reply contrib/registry/nacos/registry.go:86]: a global variable such as "var Errxxxxxx = errors.New("xxxxx")".

Comment (demoManito) [reply contrib/registry/nacos/registry.go]: > a global variable such as "var Errxxxxxx = errors.New("xxxxx")".

modified @Casper-Mars

**Review by Casper-Mars — APPROVED** [review]

**Review by haiyux — APPROVED** [review]

**Pull request merged**